docs/get-started/alerting.md

```diff
@@ -6,64 +6,161 @@
+
+Notification policies determine how Grafana alerts are routed to contact points by defining where, when, and how to send notifications.
 
 For example, you might specify a limit for the number of times a notification is sent during a certain period. This helps ensure that you don't spam your Slack channel with too many notifications about the same issue.
 
+#### PMM Root Notification policy
+
+Percona Alerting comes pre-configured with a Notification Root Policy, which is the default notification policy that PMM applies to all the alerts that don’t have a custom notification policy assigned to them. This is the **grafana-default-email** policy, which uses Email as the default contact point.
+
+#### How matching works
+
+Policies can have one or more child policies. An alert matches if the alert’s labels match all the **Matching Labels** specified on the policy.
+
+Alerts are first verified against the main policy, then against any matching child policies if the **Continue matching subsequent sibling nodes** is enabled.
+
+Alerts that don't match any specific policies are handled by the root policy. The root policy also handles any alert rules for which the assigned custom notification policy has been deleted, to ensure notifications for existing alerts continue to be delivered.
+
+#### Edit the root notification policy
+
+1. Go to {{icon.bell}} **Alerting > Notification policies** tab.
+2. Click **Edit** on the top right of the root policy box.
+3. Choose whether to keep the default Email contact point, select a new available contact point or create a new one.
+4. In the **Group by** field, specify how alert rules should be processed into notifications. If multiple alerts are matched for this policy, they will be grouped based on the labels you specify, and a notification will be sent per group.
+5. Expand the **Timing options** section and specify how notification wait times should be processed. These are short pauses the system can take to efficiently process multiple sets of alerts for notifications:
+   - **Group wait**: The default is to wait 30 seconds to buffer alerts of the same group before sending a notification initially.
+   - **Group interval**: The default is to wait five minutes before sending a batch of new alerts after the first notification was sent.
+   - **Repeat interval**: The default is to wait four hours before resending an alert.
+6. Click **Save** to save your changes.
+
+#### Create a new notification policy
+
+To create a new notification policy:
+
+1. Go to {{icon.bell}} **Alerting > Notification policies** tab.
+2. Click **New specific policy**.
+3. In the **Matching labels** section define the rules for matching alert labels. A policy will match an alert if the alert’s labels match all the matching labels specified on the policy. If there are no matchers, the policy will handle all the alert instances. For  example, you could add a **node_name=pmm-server** matcher to send out notifications only for this node.
```

Comment:
I think we need a lot more here on matchers i.e. what is the syntax if we want to use a service? Since everything flows from the matchers, I dont think giving just one example of using a node_name is sufficient (how would they even know the syntax? They might just try to put "node as I did (BTW the system accepts this and or anything else i.e. I even typed in 'funny;, even though it is WRONG...and I expect it will fail). Surprised it accepts this. 

More useful examples of possible labels and values should be here IMHO